### PR TITLE
Xml rework

### PR DESCRIPTION
**Pull request by ajl2612**

### CODE REVIEW

File: xml/schema.xsd [hunk 1]
```diff
@@ -72,21 +82,57 @@
...
+        <xsd:sequence>
+          <xsd:element name="value" type="xsd:integer" minOccurs="1" maxOccurs="1"/>
+          <xsd:element name="max" type="xsd:integer" minOccurs="0" maxOccurs="1" default=0/>
+          <xsd:element name="min" type="xsd:integer" minOccurs="0" maxOccurs="1" default=65535/>
+        </xsd:sequence>
+      </xsd:extension>
+    </xsd:complexContent>
+  </xsd:complexType>
+
+  <xsd:complexType name="signedShirtField" >
```
Comment (jaredsmithse) [inline xml/schema.xsd]: `signedShirtField` -> `signedShortField`

File: xml/schema.xsd [hunk 2]
```diff
@@ -36,7 +36,7 @@
     <xsd:complexContent>
       <xsd:extension base="varField">
         <xsd:sequence>
-          <xsd:element name="value" type="xsd:string" />
+          <xsd:element name="value" type="xsd:string" minOccurs="1" maxOccurs="1"/>
     		  <xsd:element name="pattern" type="xsd:string" minOccurs="0" maxOccurs="1"/>
```
Comment (jaredsmithse) [inline xml/schema.xsd]: Unecessary tabbing here, this doesn't seem to be from you, but can you change it for this PR?

Comment (jaredsmithse) [inline xml/schema.xsd]: Spacing as well :smiley:

**Comment by ajl2612** [comment]
Need to update example.xml and confirm validation. Then we can merge.

**Pull request merged**